src/lib/react/useRoomConnection.ts

```diff
@@ -374,10 +374,7 @@ export type RoomConnectionRef = {
     _ref: RoomConnection;
 };
 
-export default function useRoomConnection(
-    roomUrl: string,
-    roomConnectionOptions: UseRoomConnectionOptions
-): RoomConnectionRef {
+export function useRoomConnection(roomUrl: string, roomConnectionOptions: UseRoomConnectionOptions): RoomConnectionRef {
```

Comment:
This was causing webpack build errors when importing into a Create React App app.